packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -163,6 +157,11 @@ export function cloudflare(pluginConfig: PluginConfig = {}): vite.Plugin[] {
 															resolvedPluginConfig.type === "workers" &&
 															environmentName ===
 																resolvedPluginConfig.entryWorkerEnvironmentName,
+														hasNodeJsCompat:
+															resolvedPluginConfig.type === "workers" &&
+															resolvedPluginConfig.nodeJsCompatMap.has(
+																environmentName
+															),
```

Comment:
nit: re-use the helper function

```suggestion
														hasNodeJsCompat: getNodeJsCompat(environmentName) !== undefined
```

Reply:
[e43e7db](https://github.com/cloudflare/workers-sdk/pull/10482/commits/e43e7db43de161e6d1424ec6fe9842a4503adf4f)